src/dmd/expressionsem.d

```diff
@@ -6272,7 +6272,7 @@ private extern (C++) final class ExpressionSemanticVisitor : Visitor
             /* Rewrite to get rid of the comma from rvalue
              */
             if (!(cast(CommaExp)exp.e2).isGenerated)
-                exp.deprecation("Using the result of a comma expression is deprecated");
+                exp.error("The result of a comma expression can't be used");
```

Comment:
I'd prefer this to say "The result of a comma expression is deprecated and can no longer be used".  But, it's ok the way it is; I'll pull it regardless.

Reply:
Hmm it's not deprecated anymore, it's a real error and should be displayed to the user as such.

Reply:
I agree with @wilzbach.

Reply:
Would be nice to distinguish "can't be used" (due to some technical limitation) from "can't be used" (because we decided to forbid it). How about "Using the result of a comma expression is not allowed" ?